README.md

```diff
@@ -89,3 +89,26 @@ Of these variables,
 * $btn-secondary-hover-border-color
 
 are required. All the other variables are optional.
+
+To use on the devstack, you need to create `lms/static/sass/common-variables.scss`
+with the above variables, and the file `lms/static/sass/_lms-overrides.scss` that
+imports these variables using `@import 'common-variables';` and contains any other
+edx-platform-specific scss overrides.
+
+You can then install and use it as a regular theme. Make sure to run
+`make dev.static.lms` to make sure that the latest changes are picked up.
+
+# [IN DEVELOPMENT] Use as branding package for MFEs
```

Comment:
Please link the [OEP](https://open-edx-proposals.readthedocs.io/en/latest/oep-0048-brand-customization.html#decision) in this section.